.github/workflows/console-test-run.yml

```diff
@@ -154,24 +154,6 @@ jobs:
           docker load --input /tmp/console-image.tar
           docker image ls -a
 
-      #----------------------------------------------
```

Comment:
What code are we going to build below if we are removing the checkout action?

Reply:
I thought we just needed to build the trading app now that the console-test repo is within trading.

But also see that the building steps are export, 

```
- name: Build trading app
        run: |
          yarn env-cmd -f ./apps/trading/.env.stagnet1 yarn nx export trading
          DIST_LOCATION=dist/apps/trading/exported
          mv $DIST_LOCATION dist-result
          tree dist-result
```

So I imagine I will need to do a traditional build as well as?